### ISSUE TITLE
CSV upload issue

### ISSUE THREAD

**Issue by hooray4me** [churchio/onebody#540]
The CSV upload isn't working. I've tried many times in many ways. 
![image](https://cloud.githubusercontent.com/assets/12399138/9879953/3e04f848-5b8f-11e5-9a94-e2115acaeffd.png)
If I repeat the same CSV, it says 8 creates everytime...
But I cannot find the user anywhere.
![image](https://cloud.githubusercontent.com/assets/12399138/9879975/5704fdd4-5b8f-11e5-8eb0-1ff7bbe0ad4e.png)
I'm currently 6 commits behind

**Comment by hooray4me** [comment]
I still get the same result after updating to the latest commit. I had around 100 updates(email and classes) for existing users which "showed" successful in the CSV uploader, but a mysql query revealed the old values. I ended up dumping the updates into mysql with UPDATE statements, which worked. As a workaround I created the 8 new users individually thru the GUI, then updated their classes via mysql.
